packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1212,6 +1212,10 @@ oie.primaryauth.submit = Sign in
 oie.registration.form.title = Sign up
 oie.registration.form.submit = Sign Up
 oie.registration.form.update.submit = Submit
+## {0} is the form header label customized by admin
+oie.registration.form.customize.label = {0}
+## {0} is the form submit button label customized by admin
+oie.registration.form.customize.buttonLabel = {0}
```

Comment:
Since the user is setting the label in the admin side, I don't think we need to localize it any more 🤔 . On the assumption that the admin knows their audience and you're not formatting the string, we may not need this

Reply:
Are you still going to use these additions to the properties file?